scripts/postgresql-setup.sh

```diff
@@ -145,13 +153,12 @@ function dump_schema {
 
 function create_snapshot {
 	tgz_file=$1.tgz
-	dbfile=$1.sql
 	ledger_file=$2
 	tmp_dir=$(mktemp "${directory}" -t db-sync-snapshot-XXXXXXXXXX)
 	echo $"Working directory: ${tmp_dir}"
-	pg_dump --no-owner --schema=public "${PGDATABASE}" > "${tmp_dir}/$1.sql"
+	pg_dump --no-owner --schema=public --jobs="${numcores}" "${PGDATABASE}" --format=directory --file="${tmp_dir}/db/"
 	cp "$ledger_file" "$tmp_dir/$(basename "${ledger_file}")"
-	tar zcvf - --directory "${tmp_dir}" "${dbfile}" "$(basename "${ledger_file}")" | tee "${tgz_file}.tmp" \
+	tar zcvf - --directory "${tmp_dir}" "db" "$(basename "${ledger_file}")" | tee "${tgz_file}.tmp" \
```

Comment:
"--format=directory" is compressed by default no? does it make sense to re-compress ? Another way is to remove "z" from here. Only thing that is not compressed is the ledger state in this case ( hopefully eliminated one day as per your suggestion), but it also could be compressed separately. Just an idea, could save a bunch of extra CPU time potentially.

Reply:
Actually, another benefit, is that gzip is single core process, unless you use pigz for example, so while the parallelism introduced by new pg_dump is great, now you are getting slowed down by the single-threaded compression.

Reply:
I will play with this some more.

Reply:
We probably don't want to gzip the dababase files (because they are already gzipped) but probably do want to gzip the ledger state file.